developer-docs-site/docs/concepts/coin-and-token/fungible-asset.md

```diff
@@ -0,0 +1,444 @@
+---
+title: "Fungible Asset"
+id: "fungible-asset"
+---
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Fungible Asset
+
+Fungible assets (FA) are an essential part of the Aptos ecosystem, as they enable the creation and transfer of fungible
+units, which can represent different things, such as currency, shares, material in games, or any other type of asset.
+Fungible assets can be used to build decentralized applications that require a token economy, such as decentralized
+exchanges or gaming platforms.
+
```

Comment:
What do you think about adding a section to briefly compare Coin and FungibleAsset?

Reply:
We should still explain why use FungibleAssets over the existing Coin standard